src/components/Tag/Tag.scss

```diff
@@ -49,9 +45,19 @@ $icon-size: rem(16px);
       background: var(--p-surface-neutral-disabled);
       cursor: default;
       pointer-events: none;
+      color: var(--p-text-disabled);
     }
   }
 
+  &.removable {
+    padding-right: 0;
+  }
+
+  &.linkable {
+    align-items: stretch;
```

Comment:
This jumps out as a bit of an odd property to me. Can you share some info around why you need this?

Reply:
That's a great observation!! 

I went back to the playground and I don't think that's making a difference at all! 😱 

I know at some point the `a` elements were shorter in height than the tag so hovering created a weird effect where only the inner span was being highlighted. At that time I was using both `UnstyledLink` and `Stack`, which I ended up removing, so maybe that played a particular role in creating that issue!

Seeing as this is no longer needed, I'll remove it!! 🎩